packages/api/cms-api/src/warnings/warning-checker.command.ts

```diff
@@ -125,20 +125,46 @@ export class WarningCheckerCommand extends CommandRunner {
                             }
                         }
                     } else {
-                        await this.processEntityWarningsIndividually({
-                            repository,
-                            createWarnings: service.createWarnings,
```

Comment:
Maybe it would work with `createWarnings: (entity) => service.createWarnings(entity)`?

Reply:
thank you, yes it works like that!
changed in: 775a005d7d7c7974f5bd95fee25cde2c5a7d44ef